R/lint.R

```diff
@@ -151,7 +152,8 @@ reorder_lints <- function(lints) {
 #'   )
 #' }
 #' @export
-lint_dir <- function(path = ".", relative_path = TRUE, ..., exclusions = NULL, pattern = rex::rex(".", one_of("Rr"), end), parse_settings = TRUE) {
+lint_dir <- function(path = ".", relative_path = TRUE, ..., exclusions = NULL,
+                     pattern = rex::rex(".", one_of("Rr"), "" %or% "md" %or% "nw", end), parse_settings = TRUE) {
```

Comment:
This gives you a little nicer regex.

```suggestion
                     pattern =  rex::rex(".", one_of("Rr"), or("", "md", "nw"), end), parse_settings = TRUE) {
```

Reply:
Thanks, I'm not used to `rex::rex` at all :) The new one is just two characters longer than what I would produce by hand. Didn't notice how using `%or%` created two unnamed capture groups.